docs/examples/coreos-kops-tests-multimaster.md

```diff
@@ -0,0 +1,310 @@
+- Ensure our masters are deployed on 3 different AWS availability zones.
+- Ensure our nodes are deployed on 2 different AWS availability zones.
+
+
+## PRE-FLIGHT CHECK:
+
+Before rushing in to replicate this exercise, please ensure your basic environment is correctly setup. See the KOPS tutorial for more information: [KOPS Tutorial](https://github.com/kubernetes/kops/blob/master/docs/aws.md). Ensure that the following is already working in your environment:
+
+- AWS cli fully configured (aws account already with proper permissions/roles needed for kops). Depending on your distro, you can setup directly from packages, or if you want the most updated version, use "pip" and installed by issuing a "pip install awscli" command. Your choice !.
+- Local ssh key ready on ~/.ssh/id_rsa / id_rsa.pub. You can generate it using "ssh-keygen" command: `ssh-keygen -t rsa -f ~/.ssh/id_rsa -P ""`
+- Region set to us-east-1 (az's: us-east-1a, us-east-1b, us-east-1c, us-east-1d and us-east-1e). For this exercise we deployed our cluster on US-EAST-1. If you want to use another region, feel free to change it. Just ensure to have at least 3 availability zones.
+- kubectl and kops installed. For this last part, you can do this with the following commnads (do this as root please). Next commands asume you are running a amd64/x86_64 linux distro:
+
+```bash
+cd ~
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+wget https://github.com/kubernetes/kops/releases/download/1.7.0/kops-linux-amd64
+chmod 755 kubectl kops-linux-amd64
+mv kops-linux-amd64 kops
+mv kubectl kops  /usr/local/bin
+```
+
+
+## AWS/KOPS ENVIRONMENT INFORMATION SETUP:
+
+First, using some scripting and asuming you already configured your "aws" environment on your linux system, use the following commands in order to export your AWS access/secret:
+
+```bash
+export AWS_ACCESS_KEY_ID=`grep aws_access_key_id ~/.aws/credentials|awk '{print $3}'`
+export AWS_SECRET_ACCESS_KEY=`grep aws_secret_access_key ~/.aws/credentials|awk '{print $3}'`
+echo "$AWS_ACCESS_KEY_ID $AWS_SECRET_ACCESS_KEY"
+```
+
+Create a bucket (if you don't already have one) for your cluster state:
+
+```bash
+aws s3api create-bucket --bucket kops-tigerlinux-cluster-state --region us-east-1
+```
+
+Then export the name of your cluster along to the "S3" URL of your bucket:
+
+```bash
+export NAME=coreosbasedkopscluster.k8s.local
+export KOPS_STATE_STORE=s3://kops-tigerlinux-cluster-state
+```
+
+
+## COREOS IMAGE INFORMATION:
+
+CoreOS webpage includes a "json" with the updated list of lattest images: https://coreos.com/dist/aws/aws-stable.json
+
+If you install the "jq" utility (available on most distros) you can obtain the "ami" for a specific region:
+
+
+```bash
+curl -s https://coreos.com/dist/aws/aws-stable.json|sed -r 's/-/_/g'|jq '.us_east_1.hvm'|sed -r 's/_/-/g'
```

Comment:
Yup. As a matter of give options to "skin a cat" to the end-user, I've included both ways on the documentation.